infra-provider-kafka/src/test/java/com/expediagroup/streamplatform/streamregistry/provider/impl/KafkaInfraManagerTest.java

```diff
@@ -107,7 +108,7 @@ public void testUpsertTopicsForExistingStreamWithMatchingConfig() throws StreamC
         // Mock it as an existing topic
         when(AdminUtils.topicExists(zkUtils, TOPIC)).thenReturn(true);
 
-        KafkaInfraManager kafkaManagerSpy = spy(kafkaManager);
+        KafkaInfraManager kafkaManagerSpy = PowerMockito.spy(kafkaManager);
```

Comment:
I see in a few places where static imports have been switched either this way around or the other (i.e. to no longer have the full name), not really too bothered as long as it works, just feels like it doesn't necessarily need to be in this PR but NABD either way.

Reply:
Yea agree. Intellij did this by itself. I'll remove the ones we spot.